src/features/ExtensionCommands.ts

```diff
@@ -408,6 +423,53 @@ export class ExtensionCommandsFeature implements IFeature {
         return promise;
     }
 
+    private normalizeFilePath(filePath: string): string {
+        var platform = os.platform();
+        if (platform == "win32") {
+            // Make sure the file path is absolute
+            if (!path.win32.isAbsolute(filePath))
+            {
+                filePath = path.win32.resolve(
+                    vscode.workspace.rootPath,
+                    filePath);
+            }
+
+            // Normalize file path case for comparison for Windows
+            return filePath.toLowerCase();
+        } else {
+            // Make sure the file path is absolute
+            if (!path.isAbsolute(filePath))
+            {
+                filePath = path.resolve(
+                    vscode.workspace.rootPath,
+                    filePath);
+            }
+
+            //macOS is case-insensitive
+            if (platform == "darwin") {
+                filePath = filePath.toLowerCase();
+            }
+
+            return  filePath;
+        }
+    }
+
+    private findTextDocument(filePath: string): boolean {
+        // since Windows and macOS are case-insensitive, we need to normalize them differently
+        var canFind = vscode.workspace.textDocuments.find(doc => {
+            var docPath, platform = os.platform();
+            if (platform == "win32" || platform == "darwin") {
+                // for Windows and macOS paths, they are normalized to be lowercase
+                docPath = doc.fileName.toLowerCase();
+            } else {
+                docPath = doc.fileName;
+            }
+            return docPath == filePath;
```

Comment:
Probably want to use a `===` here to prevent any type coercion although both should be strings.  I tend to favor `===` and `!==` unless I know I want the comparison to do some type coercion for me.

Reply:
fixed!